packages/generator/package.json

```diff
@@ -68,7 +69,8 @@
     "react": "18.2.0",
     "typescript": "^4.8.4",
     "unbuild": "0.6.9",
-    "watch": "1.0.2"
+    "watch": "1.0.2",
+    "zod": "3.20.2"
```

Comment:
having zod as a devDep causes global use of blitz to fail because zod isn't install there.

We need to move this to a regular dep but with version range that supports all of v3, so we don't conflict with user's zod version.


```
/tmp> blitz new blitz-fields
Error: Cannot find module 'zod'
Require stack:
- /Users/b/Library/pnpm/global/5/.pnpm/@blitzjs+generator@0.0.0-multiple-fields-20230126103232/node_modules/@blitzjs/generator/dist/index.cjs
- /Users/b/Library/pnpm/global/5/.pnpm/blitz@0.0.0-multiple-fields-20230126103232/node_modules/blitz/dist/chunks/new.cjs
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/b/Library/pnpm/global/5/.pnpm/@blitzjs+generator@0.0.0-multiple-fields-20230126103232/node_modules/@blitzjs/generator/dist/index.cjs:15:11)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/b/Library/pnpm/global/5/.pnpm/@blitzjs+generator@0.0.0-multiple-fields-20230126103232/node_modules/@blitzjs/generator/dist/index.cjs',
    '/Users/b/Library/pnpm/global/5/.pnpm/blitz@0.0.0-multiple-fields-20230126103232/node_modules/blitz/dist/chunks/new.cjs'
  ]
}
```

Reply:
let me know when you publish a new version to test

Reply:
> let me know when you publish a new version to test

@flybayer I fixed the issue and published